fintech-examples/fintech-example-tests/src/test/java/de/adorsys/fintech/tests/e2e/steps/FintechStagesUtils.java

```diff
@@ -9,16 +9,20 @@
 
     public static final String X_REQUEST_ID = "X-REQUEST-ID";
     public static final String PIN = "1234";
-    public static String USERNAME = "bob";
-    public static String FINTECH_LOGIN = "tom";
+    public static String OBA_USERNAME = "htc";
```

Comment:
Instead of using OBA_USERNAME as `htc` you need to use the user you have created with 
`.create_new_user_in_sandbox_tpp_management(username, PIN)`